build.sbt

```diff
@@ -443,11 +451,25 @@ lazy val scaldingParquetScrooge = module("parquet-scrooge")
         exclude("com.twitter.elephantbird", "elephant-bird-core"),
        "com.twitter" %% "scrooge-serializer" % scroogeVersion,
       "org.apache.hadoop" % "hadoop-client" % hadoopVersion % "provided",
-      "com.novocode" % "junit-interface" % "0.11" % "test",
       "junit" % "junit" % junitVersion % "test"
+    )
+).dependsOn(scaldingParquetCascading % "compile->compile;test->test", scaldingParquetScroogeFixtures % "test->test")
 
+lazy val scaldingParquetScrooge = module("parquet-scrooge")
```

Comment:
shouldn't we merge this with `scaldingParquetScroogeCascading`? Same dependencies, same use case, right?